### PR TITLE
Add Conda packaging

### PR DESCRIPTION
**Pull request by danielsuo**
Create Conda packages and test in Travis. A few outstanding issues

- [ ] Update documentation (README, website, docs)
- [ ] Understand why Linux build is hanging on ```test_brsa.py``` for Travis, but not anywhere else [link](https://travis-ci.org/danielsuo/brainiak/jobs/336390600)
- [ ] Confirm whether users need ```apt install build-essential``` or ```xcode-select install```
- [ ] Upload to anaconda.org

Unrelated issues: code coverage (see #327 and #326).

### CODE REVIEW

File: .conda/bld.bat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+:: NOTE: Windows not supported!
```
Comment (mihaic) [inline .conda/bld.bat]: Is this script necessary?

**Comment by danielsuo** [comment]
Don't believe so; it was automatically generated. I'm testing now whether
we need it to build.

**Comment by danielsuo** [comment]
Ok, initially looks good; not sure about code coverage or hanging tests though. It'll be good to get wider user testing before we make a more public change.

**Comment by mihaic** [comment]
@danielsuo, can you please add to CONTRIBUTING instructions for uploading Conda package?

**Comment by danielsuo** [comment]
Sounds good; should this go in ```Making a release```?

**Comment by mihaic** [comment]
Yes.

**Comment by danielsuo** [comment]
I was unsure of how to organize the process since there's a for loop
involved for OS / python versions, but put something up.

**Comment by mihaic** [comment]
I propose we update the website after making the release.

As far as I understand, we do not need `build-essential` on Linux after switching to using the Conda compilers:
https://conda.io/docs/user-guide/tasks/build-packages/compiler-tools.html

The MacOS requirements situation is less clear to me. @danielsuo, if you have access to a MacOS VM, can you try installing the package without first installing Xcode? I uploaded one to my channel:
https://anaconda.org/mihaic/brainiak

With the new compilers all tests are passing.

Finally, we will upload to the `brainiak` channel after the release.

**Pull request merged**